spec/requests/carto/api/public/federated_tables_controller_spec.rb

```diff
@@ -7,20 +7,62 @@
 
   before(:all) do
     host! "#{@user1.username}.localhost.lan"
+
+    puts "Starting remote server"
+    @dir = Cartodb.get_config(:federated_server, 'dir')
+    port = Cartodb.get_config(:federated_server, 'port')
+    user = Cartodb.get_config(:federated_server, 'test_user')
+    pg_bindir = Cartodb.get_config(:federated_server, 'pg_bindir_path')
+    unless pg_bindir.present?
+      pg_bindir = `pg_config --bindir`.delete!("\n")
+    end
+    @pg_ctl     = "#{pg_bindir}/pg_ctl"
+    @psql       = "#{pg_bindir}/psql"
+
+    unless @dir.present?
+      raise "Federated server directory is not configured!"
+    end
+    unless port.present?
+      raise "Federated server port is not configured!"
+    end
+    unless user.present?
+      raise "Federated server user is not configured!"
+    end
+    system("which #{@pg_ctl}") || raise("Federated server pg_ctl could not be found")
+    system("which #{@psql}") || raise("Federated server psql could not be found")
+
+    puts "Starting remote server"
+    system("#{@pg_ctl} start --silent -D #{@dir} >/dev/null") || raise("Could not start the federated DB")
+
+    @remote_host     = "127.0.0.1"
+    @remote_port     = "#{port}"
+    @remote_database = "#{user}"
```

Comment:
```suggestion
    @remote_database = user
```
Should work, shouldn't it?